src/utilities/get-adjacent-pages/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function getAdjacentPages(haystack, needle, by = 'url') {
```

Comment:
```suggestion
export default function getAdjacentPages(haystack, needle, by = 'url') {
```

Reply:
Updated. I had to update `.babelrc` file too for test.